processing/data_collection/gazette/spiders/rn_mossoro.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, date
+
+import dateutil
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMossoroSpider(BaseGazetteSpider):
+
+    name = "rn_mossoro"
+    allowed_domains = ["jom.prefeiturademossoro.com.br"]
+
+    TERRITORY_ID = "2408003"
+
+    def __init__(self, start_date=None, end_date=None):
```

Comment:
I think you need to get `*args` and `**kwargs` here.
In case we set `-a spider_arg=123` in the CLI, they are set to `__init__()` as `**kwargs` and set as class attributes by the base spider.

Also, I'd expect the base class to set `start_date` and `end_date` properly, and only set them here in case we had to overwrite the value

Reply:
Fixed.